.doc_gen/metadata/cross_metadata.yaml

```diff
@@ -612,3 +612,53 @@ cross_MessagesTopicsAndQueues:
   services:
     sns:
     sqs:
+cross_ResilientService:
```

Comment:
Yes, the category is what matters for where it's published, so this will appear under scenarios in the code examples chapters. I put the metadata in cross_metadata.yaml because it didn't fit anywhere else and I didn't want to create a yet another yaml file. 😅